js/apps/account-ui/src/root/Root.tsx

```diff
@@ -57,9 +61,10 @@ export const Root = () => {
             features={{ hasManageAccount: false }}
             showNavToggle
             brand={{
-              src: joinPath(environment.resourceUrl, "logo.svg"),
+              src: joinPath(environment.resourceUrl, brandImage),
               alt: t("logo"),
               className: style.brand,
+              onClick: environment.logoUrl ? brandOnClickHandler : undefined,
```

Comment:
This should be handled as a link, using JavaScript to navigate breaks certain user expectations and hinders accessibility.

Reply:
Right.  That's what I was asking Erik about.  Needs to be done in the masthead, IMO.